public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -78,9 +89,17 @@ export class LibraryVizPanel extends SceneObjectBase<LibraryVizPanelState> {
 function getLoadingPanel(title: string) {
   return new VizPanel({
     title,
+    $data: new SceneDataTransformer({ transformations: [], $data: new SceneQueryRunner({ queries: [] }) }),
     menu: new VizPanelMenu({
       $behaviors: [panelMenuBehavior],
     }),
+    titleItems: [
+      new VizPanelLinks({
+        rawLinks: [],
+        menu: new VizPanelLinksMenu({ $behaviors: [panelLinksBehavior] }),
+      }),
+      new PanelNotices(),
+    ],
```

Comment:
We're expecting this to exist on panels. When the PanelEditor activates, the library panel does not necessarily have the panel loaded. This means we have to add these to this place holder panel.

Reply:
@oscarkilhed but this was fixed by waiting for the library panel to load before entering panel edit right?

Reply:
Yes!